tests/github-retag-and-push.bats

```diff
@@ -11,45 +11,47 @@ teardown () {
 }
 
 
-@test "valid other branch" {
+@test "misc branch, shorten sha" {
     export GITHUB_REF=refs/heads/foobar
-    export GITHUB_SHA=12345678
-    run docker tag busybox libero/otherbranch:12345678
-    run github/retag-and-push.sh otherbranch 12345678
+    export GITHUB_SHA=shortsha-butwithlongbitpresent
+    run docker tag busybox libero/otherbranch:123
+    run github/retag-and-push.sh otherbranch 123
     [ "$status" -eq 0 ]
-    run docker pull ${DOCKER_REGISTRY}liberoadmin/otherbranch:foobar-12345678
+    run docker pull "${DOCKER_REGISTRY}liberoadmin/otherbranch:foobar-shortsha"
     [ "$status" -eq 0 ]
     run docker pull ${DOCKER_REGISTRY}liberoadmin/otherbranch:latest
     [ "$status" -eq 1 ]
 }
 
-@test "valid master tagging" {
+@test "valid master tagging w/wo timestamp" {
     export GITHUB_REF=refs/heads/master
-    export GITHUB_SHA=12345678
-    run docker tag busybox libero/my-dummy-project:12345678
-    run github/retag-and-push.sh my-dummy-project 12345678
+    export GITHUB_SHA=1234567890ab
```

Comment:
shoudldn't these values be 8-char long to mimic realistic ones?

Reply:
I've switched them to realistic full length hashes in the hope that this also helps clarify that the hash the shortened by the script. the GITHUB_SHA env var is always the full hash.